xbmc/FileItem.cpp

```diff
@@ -2870,289 +2873,225 @@ void CFileItemList::Stack(bool stackFiles /* = true */)
   // items needs to be sorted for stuff below to work properly
   Sort(SortByLabel, SortOrderAscending);
 
-  StackFolders();
-
-  if (stackFiles)
-    StackFiles();
```

Comment:
Before this PR, stacking of files was done only conditionally, now always. Have you thought about possible performance problems? Just asking.

Reply:
As far as I can see stackFiles was always true - every call was just Stack() so defaulted to true.